pkg/cri/config/config_test.go

```diff
@@ -383,6 +383,20 @@ func TestValidateConfig(t *testing.T) {
 			},
 			expectedErr: "`privileged_without_host_devices_all_devices_allowed` requires `privileged_without_host_devices` to be enabled",
 		},
+		"invalid drain_exec_sync_io_timeout input": {
+			config: &PluginConfig{
+				ContainerdConfig: ContainerdConfig{
+					DefaultRuntimeName: RuntimeDefault,
+					Runtimes: map[string]Runtime{
+						RuntimeDefault: {
+							Type: "default",
+						},
+					},
+				},
+				DrainExecSyncIOTimeout: "10",
+			},
+			expectedErr: "invalid drain exec sync io timeout: time: missing unit in duration \"10\"",
```

Comment:
Matching against the entire error (meaning including the bit from the stdlib) seems brittle, but I don't know how much the Go team values not changing random error strings 🤷‍♂️. It's just an errors.New defined inline in the function so if they ever changed the text a little this would fail https://cs.opensource.google/go/go/+/refs/tags/go1.20.1:src/time/format.go;l=1652

Reply:
good point! just verify the error should contains "invalid `drain_exec_sync_io_timeout`".